salt/modules/parted_partition.py

```diff
@@ -542,39 +545,6 @@ def mkpart(device, part_type, fs_type=None, start=None, end=None):
     return out
 
 
-def mkpartfs(device, part_type, fs_type, start, end):
```

Comment:
We cannot remove this function, because a user could be using this even if its not being called from another state/execution module. Could you possibly have this function call the other to ensure backwards compatibility?

Reply:
Of course. I will do that tonight.

Reply:
Re-added the mkpartfs which now is just a call to mkpart. Also added a test to check if all arguments from mkpartfs are reaching mkpart.